ansible/roles/superhero-dashboard/templates/configmap.yml.j2

```diff
@@ -13,3 +13,4 @@ data:
   FEATURE_CTL_TOOLS_ENABLED: "{{ FEATURE_CTL_TOOLS_ENABLED|default(false, true) }}"
   FEATURE_LTI_TOOLS_ENABLED: "{{ FEATURE_LTI_TOOLS_ENABLED|default(true, true) }}"
   FEATURE_SCHOOL_SANIS_USER_MIGRATION_ENABLED: "{{ FEATURE_SCHOOL_SANIS_USER_MIGRATION_ENABLED|default(false, true) }}"
+  FEATURE_SHOW_OUTDATED_USERS: "{{ FEATURE_SHOW_OUTDATED_USERS|default(false, true) }}"
```

Comment:
I don't know if it can be confusing if the SchoolFeature and the feature flag to show the OutdatedUsers configuration are named very similar. Maybe another name for the feature flag would be better like FEATURE_OUTDATED_USERS_CONFIGURATION